app.py

```diff
@@ -4,15 +4,14 @@
 from resources.quote import QuotesApi, QuoteApi
 from resources.author import AuthorsApi, AuthorApi
 from dotenv import load_dotenv, find_dotenv
+from utils.logging import init_logger
 import logging
 import os
 
 load_dotenv(find_dotenv())
 
-# Logging Configuration
-logging.basicConfig(filename='logs.log',
-                level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(name)s - %(message)s')
-
+logger = logging.getLogger("app")
+init_logger(logger, os.environ.get("LOG_FILE"))
```

Comment:
You can avoid passing LOG FILE as param here and use the constant directly in logger.py as it is an ENV variable

Reply:
sure